lib/generators/gql/gql_generator_base.rb

```diff
@@ -37,5 +37,32 @@ def map_model_types(model_name)
           .map { |col| {name: col.name, gql_type: type_map[col.type]} }
       end
     end
+
+    def root_directory(namespace)
+      "app/graphql/#{namespace.underscore}"
+    end
+
+    def wrap_in_namespace(namespace)
+      namespace = namespace.split('::')
+      namespace.shift if namespace[0].empty?
+
+      code = namespace.each_with_index.map { |name, i| "  " * i + "module #{name}" }.join("\n")
```

Comment:
There's the `indent` function but it's only supported in Rails > 5.1 and you (nor GQL Ruby) has such a restriction.

Reply:
Are you saying rails 5.1 has a generator helper function that helps with properly indenting generator files? If so, I'm fine with this lib only supporting rails 5.1+.

Reply:
> Are you saying rails 5.1 has a generator helper function that helps with properly indenting generator files? 

5.2. But it more or less does what I just did. I'd leave it. You get < 5.2 support for free with minimal overhead, I think.